lib/PuppeteerSharp/Session.cs

```diff
@@ -152,7 +148,7 @@ internal void OnMessage(string message)
             }
         }
 
-        internal void Close()
+        internal void OnClosed()
```

Comment:
renamed to match - https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/lib/Connection.js#L212

Reply:
Awesome